packages/codecs/src/known/common.ts

```diff
@@ -42,4 +42,9 @@ export type Option<T> = T | undefined;
 
 export type ResultPayload<Ok, Err> = $.ResultPayload<Ok, Err>;
 
-// TODO add typeIn for other primitives codecs!
+export type u8 = number;
+export type u16 = number;
+export type u32 = number;
+export type u64 = bigint;
+export type u128 = bigint;
+export type u256 = bigint;
```

Comment:
There are also other primitive codecs to define types as well, you can check the list [here](https://github.com/CoongCrafts/delightfuldot/blob/3eec66ce2ef656dd3963d72a01f2a121dc95a8ab/packages/shape/src/extension/shape.ts#L72-L85)